python/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+
+IF(NOT Kokkos_ENABLE_PYBIND11)
+    return()
+ENDIF()
+
+
+# don't build a static python module
+SET(BUILD_SHARED_LIBS ON)
+
+# pybind11 has not migrated to CMAKE_CXX_STANDARD and neither has kokkos
+if(NOT Kokkos_CXX_STANDARD)
+    set(Kokkos_CXX_STANDARD 11)
+endif()
+IF(MSVC)
+    set(PYBIND11_CPP_STANDARD /std:c++${Kokkos_CXX_STANDARD} CACHE STRING "C++ Language standard used by Pybind11")
+ELSE()
```

Comment:
Is this something weird that `PYBIND11_CPP_STANDARD` has to be defined before the `find_package` call?

Reply:
No, it's not a hard requirement, just ensuring that we are using the same standard. We could probably get away with pybind choosing the standard if you really want to